src/patternfly/components/VerticalNav/styles.scss

```diff
@@ -0,0 +1,134 @@
+@import "../../patternfly-utilities";
+
+.pf-c-vertical-nav {
+  /* stylelint-disable */
+
+  // Vertical nav
+  --pf-c-vertical-nav--Color:             var(--pf-global--BackgroundColor--dark-100);
+  --pf-c-vertical-nav--BackgroundColor:   var(--pf-global--BackgroundColor--light-100);
+
+  // Vertical nav link
+  --pf-c-vertical-nav__link--PaddingTop:      var(--pf-global--spacer--lg);
+  --pf-c-vertical-nav__link--PaddingRight:    calc(var(--pf-global--spacer--lg) + .875rem + var(--pf-c-vertical-nav__link__arrow--Right));
+  --pf-c-vertical-nav__link--PaddingBottom:   var(--pf-global--spacer--lg);
+  --pf-c-vertical-nav__link--PaddingLeft:     var(--pf-global--spacer--xl);
+  --pf-c-vertical-nav__link--MarginTop:       0;
+  --pf-c-vertical-nav__link--MarginBottom:    0;
+  
+  // Vertical nav link active :after
+  --pf-c-vertical-nav__link--after--active--Left:    calc(var(--pf-c-vertical-nav__link--PaddingLeft) + 1.5rem + var(--pf-c-vertical-nav__link__icon--MarginRight));
+  --pf-c-vertical-nav__link--after--active--Width:   2rem;
```

Comment:
Can you please add a comment abut this magic values? I understand why they are here, but I wonder if people would.

Reply:
Removing the magic values

Reply:
not need to remove them, just make a comment to explain what they are doing

Reply:
sorry, replying on wrong comment